src/data/finset/noncomm_prod.lean

```diff
@@ -226,4 +226,20 @@ begin
   { simp [ha, IH] }
 end
 
+@[to_additive] lemma noncomm_prod_union_of_disjoint [decidable_eq α] {s t : finset α}
```

Comment:
```suggestion
/- The non-commutative version of `finset.prod_union` -/
@[to_additive /-" The non-commutative version of `finset.sum_union` "-/]
lemma noncomm_prod_union_of_disjoint [decidable_eq α] {s t : finset α}
```
The difference in names is fine, but lets add a reference back to the other name.